ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -124,10 +128,11 @@ export default class ConfirmAddSuggestedToken extends Component {
               type="default"
               large
               className="page-container__footer-button"
-              onClick={() => {
-                removeSuggestedTokens().then(() =>
-                  history.push(mostRecentOverviewPage),
-                );
+              onClick={async () => {
+                await suggestedAssets.forEach(async ({ id }) => {
```

Comment:
I wasn't aware you could do this on a `forEach` ; maybe a `Promise.all`+`suggestedAssets.map` would be more readable?

Reply:
This does work but happy to make that change if you'd prefer!

Reply:
ping here @darkwing 😅

Reply:
the await isn't doing anything here. The only reason it works is that the iterator is an async and all the logic is done there. Map + promise.all is a cleaner way of doing this.